### ISSUE TITLE
Windows Ditto App Pasting Holds CTRL+SHIFT

### ISSUE THREAD

**Issue by fohrums** [jackrosenthal/threelayout#18]
I'm using 3L Keyboard Layout on Windows via Autohotkey Script.

While using 3L Keyboard Layout on Windows 10 I get a problem with the clipboard application `Ditto.` Problem is when using *Ditto* to paste causes Windows to stay at holding CTRL+SHIFT (*or perhaps maybe just shift only*) and locks up keyboard with no way to type any other key. Only restarting PC restores the keyboard.

I know it holds Shift atleast because when clicking on files it highlights them and clicking on the already highlighted files will de-select them therefore not highlighting them at that point.

This is regardless of running Ditto as Admin and/or disabling the Ditto preference `Send Paste (Otherwise it will just load the item on the clipboard)`

**How to use 3L Keyboard Layout ahk script on Windows while simultaneously using the clipboard application Ditto without the keyboard locking up?**

**Comment by jackrosenthal** [comment]
Hi, I'll be sure to take a look at this in the next few days. There's a good chance the issue is upstream in AutoHotKey though.

**Comment by jackrosenthal** [comment]
I created an AHK script containing just this:

```
v::d
```

This should remap the ``v`` key to ``d``.

And when I pasted using the "ditto" tool into a Firefox window, Firefox bookmarked the page instead of pasting the text (indicating that ``Ctrl-D`` was pressed).

So it appears that the issue is that Ditto is pressing ``Ctrl + <key physically located where V is in the QWERTY layout>`` in order to paste.

So this appears to be a bug with Ditto. I'd report the bug there.
